tee-worker/core-primitives/stf-primitives/src/error.rs

```diff
@@ -24,54 +24,64 @@ pub type StfResult<T> = Result<T, StfError>;
 
 #[derive(Debug, Display, PartialEq, Eq, Encode, Decode, Clone)]
 pub enum StfError {
-	#[codec(index = 0)]
-	#[display(fmt = "Insufficient privileges {:?}, are you sure you are root?", _0)]
-	MissingPrivileges(AccountId),
-	#[codec(index = 1)]
-	#[display(fmt = "Valid enclave signer account is required")]
-	RequireEnclaveSignerAccount,
-	#[codec(index = 2)]
-	#[display(fmt = "Error dispatching runtime call. {:?}", _0)]
-	Dispatch(String),
-	#[codec(index = 3)]
-	#[display(fmt = "Not enough funds to perform operation")]
-	MissingFunds,
-	#[codec(index = 4)]
-	#[display(fmt = "Invalid Nonce {:?} != {:?}", _0, _1)]
-	InvalidNonce(Nonce, Nonce),
-	#[codec(index = 5)]
-	StorageHashMismatch,
-	#[codec(index = 6)]
-	InvalidStorageDiff,
-	#[codec(index = 7)]
-	InvalidMetadata,
 	// litentry
-	#[codec(index = 8)]
+	#[codec(index = 0)]
```

Comment:
Because they keep adding new entries which will break the litentry entries anyway (you could append them only, but it's a bit awful in terms of readability)

So I made the same changes as what we did to `TrustedCall` enum - put litentry's entries at front so that future upstream changes won't change the codec index